test/data/config.json

```diff
@@ -413,7 +413,7 @@
 				"secured": false
 			},
 			"externalAuth": {
-				"url": "https://httpbin.org/basic-auth/user/passwd",
+				"url": "https://httpbun.com/basic-auth/user/passwd",
```

Comment:
we should probably use the internal httpbun now, with the NIP ip

Reply:
yeah, but if you create httpbun internal service as part of the framework (like httpbun.something.svc.cluster.local) you can call it from here as it is a valid domain for Kubernetes

Reply:
fair, id have to dig into the framework a bit more on how to do that :sweat_smile:

Reply:
Should we do this as part of another PR?